install.sh

```diff
@@ -306,7 +306,7 @@ nvm_do_install() {
     command printf "${SOURCE_STR}"
   else
     case "${NVM_PROFILE-}" in
-      ".bashrc" | ".bash_profile" | ".zshrc")
+      *"/.bashrc" | *"/.bash_profile" | *"/.zshrc")
```

Comment:
How did this work in your original testing? Can we add "install script" tests that would have caught this?